hack/e2e/eks-cluster.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: eksctl.io/v1alpha5
```

Comment:
I think this file needs to be called `hack/e2e/eks-cluster.yaml.template`.
Checking...

Reply:
Confirmed. Fixing.